vimms/MassSpec.py

```diff
@@ -538,10 +538,22 @@ def _get_scan(self, scan_time, params):
         scan_mzs = np.array(scan_mzs)
         scan_intensities = np.array(scan_intensities)
 
+        #added condition for checking collision energy of scan
+        #collision energy>0, specify and no isolation, return MS2 data in an MS1 scan
+        collision_energy = params.get(ScanParameters.COLLISION_ENERGY)
+        if params.get(ScanParameters.ISOLATION_WINDOWS) is None:
+            specified_iso_windows = False #MS2 scan: pick a precursor m/z and isolation window
+        else:
+            specified_iso_windows = True #MS1 fragmentation scan: specified m/z range to measure fragments e.g. AIF
```

Comment:
Not sure on the logic here?
For AIF, I don't think we would ever specify an isolation window. Happy for the code to admit that possibility though.
So, is the default method for making MS1 fragmentation scans to:
- set collision energy >0
- set isolation window as the whole m/z range?
That would be fine, I think

Reply:
At the moment, it looks like if the isolation is not specified then it will not do fragmentation. This is fine, but needs to be very clear in the docs.

Reply:
yes, I was thinking making ms1 fragmentation while both collision energy> 0 and set isolation window as whole m/z range. That keep dda (without isolation window specified) not being impacted.

Reply:
I don't think the comments are quite right are they? Perhaps just remove -- the code makes sense without. i.e. if specified iso window = False, it doesn't necessarily mean an MS2 scan?